src/seedu/addressbook/Main.java

```diff
@@ -81,6 +83,14 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
+            try {
+            	FileReader reader = new FileReader(storage.getPath());
+            }
+            catch (FileNotFoundException fnfe) {
+            	ui.showToUser(fnfe.getMessage());
+                System.exit(0);
```

Comment:
As mentioned in the tutorial, this was a good attempt at handling an exception. It's good that the error message is thrown to the UI to handle, but throwing the error message this early means that the data is not saved, and the file is not recovered. Furthermore, the exception would be thrown during the program's initial launch (even before a file has a chance to be created), and the program would be forcefully terminated, rendering the program unusable.